test/Microsoft.ML.Tests/LearningPipelineTests.cs

```diff
@@ -42,5 +44,40 @@ public void CanAddAndRemoveFromPipeline()
             pipeline.Add(new Trainers.StochasticDualCoordinateAscentRegressor());
             Assert.Equal(3, pipeline.Count);
         }
+
+        class InputData
+        {
+            [Column(ordinal: "1")]
+            public string F1;
+        }
+
+        class TransformedData
+        {
+            [ColumnName("F1")]
+            public float[] TransformedF1;
+        }
+
+        [Fact]
+        public void TransformOnlyPipeline()
+        {
+            const string _dataPath = @"..\..\Data\breast-cancer.txt";
+            var pipeline = new LearningPipeline();
+            pipeline.Add(new TextLoader<InputData>(_dataPath, useHeader: false));
+            pipeline.Add(new CategoricalHashOneHotVectorizer("F1") { HashBits = 10, Seed = 314489979, OutputKind = CategoricalTransformOutputKind.Bag });
```

Comment:
Is this issue only affecting transforms only pipeline?

Reply:
yeah